magicblock-processor/src/scheduler/coordinator.rs

```diff
@@ -151,10 +151,10 @@ impl ExecutionCoordinator {
         transaction: &TransactionWithId,
     ) -> Result<(), BlockerId> {
         let message = transaction.txn.transaction.message();
-        let accounts_to_lock = message.account_keys().iter().enumerate();
+        let accounts_to_lock = message.account_keys().iter();
```

Comment:
don't see, how this style change is related to the PR, and it's not any way more idiomatic or succinct